Makefile

```diff
@@ -26,6 +24,12 @@ GO ?= go
 CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
 GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
 
+GO_BUILD=$(GO) build
+# Go module support: set `-mod=vendor` to use the vendored sources
+ifeq ($(shell go help mod >/dev/null 2>&1 && echo true), true)
+  GO_BUILD=GO111MODULE=on $(GO) build -mod=vendor
```

Comment:
@vrothberg What is the reason to still keep `GO111MODULE=on`?

Reply:
The Makefile still turns it off to avoid any potential side-effect of other tools `go get`-ing around, so we need to explicitly turn it on.

Reply:
I see, thanks for the clarification. Some dot releases of go1.11 will not work with that but I think we can live with that (the latest one should work definitely). :+1: